libraries/node-core-library/src/RealNodeModulePath.ts

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as fs from 'node:fs';
```

Comment:
Why not `FileSystem` since we're already here?

Reply:
Because that increases the footprint of this module substantially at runtime. Right now in Jest I deep import just this module, which has no dependencies.

Reply:
Do we not import other stuff from NCL in the Jest test runtime already?

Reply:
Also because I stub out all of the file system calls for unit testing or for injection of custom file systems (e.g. for webpack), so it is more standard to mock the contract of node's builtin `fs` module.